homeassistant/components/microsoft/tts.py

```diff
@@ -64,12 +65,22 @@
     "zh-tw",
 ]
 
+OUTPUTS = {
+    "wav-8": ("wav", "riff-8khz-16bit-mono-pcm"),
+    "wav-16": ("wav", "riff-16khz-16bit-mono-pcm"),
+    "wav-24": ("wav", "riff-24khz-16bit-mono-pcm"),
+    "mp3-16": ("mp3", "audio-16khz-128kbitrate-mono-mp3"),
+    "mp3-24": ("mp3", "audio-24khz-160kbitrate-mono-mp3"),
```

Comment:
I'm currently (localy testing) using an additional audio option, which has even better sound and that is wav-48 (audio-48khz-196kbitrate-mono-mp3). Can you please also add this one to the available options, even if it might not be supported by Google Cast devices - it sure works for me over a 3.5 audio jack.